translations/team.json

```diff
@@ -85,19 +85,19 @@
t_on_the_state_research_program_technical_diagnostics_5_4_19_detection_of_informative'>[27]</a>, <a href='#annual_report_on_the_state_research_program_technical_diagnostics_5_4_19_detection_of_informative'>[28]</a>, <a href='#description_of_the_ethernet_mac_lite_module_on_vhdl_on_xilinx_fpga_s_yu_vasyukevich'>[31]</a>, <a href='#realisation_of_the_tract_of_dvb-s_transmitter_pretreatment_based_on_fpga_s_yu_vasyukevich_a'>[34]</a>, <a href='#information_technologies_and_their_influence_on_the_future_of_humankind_i_g_davydov_a_v'>[35]</a>, <a href='#methods_of_frequency_and_time_analysis_of_a_set_of_empirical_modes_of_vibration_signals'>[36]</a>, <a href='#overview_of_adaptive_algorithms_for_defect_classification_of_engineering_products_i_g_davydov_a_v'>[37]</a>, <a href='#overview_of_adaptive_algorithms_for_vibration_fault_detection_for_rolling_bearings_i_g_davydov_a'>[38]</a>, <a href='#applicability_of_modern_mobile_phones_to_the_problems_of_vibration_diagnostics_a_v_tsurko_s'>[40]</a>, <a href='#development_of_methodological_literature_on_laboratory_work_for_students_of_technical_majors_i_g_davydov'>[41]</a>, <a href='#adaptive_algorithms_in_vibration_diagnosis_i_g_davydov_a_v_tsurko_materials_of_the_49th'>[42]</a>, <a href='#shock_pulse_repetition_frequency_estimation_algorithm_based_on_periodic_wavelet_transform_i_g_davydov_s'>[43]</a>, <a href='#laboratory_workshop_on_vsr_and_pkipptu_courses_i_g_davydov_a_v_tsurko_s_yu'>[44]</a>, <a href='#the_line_of_development_of_the_social_paradigm_with_a_systematic_approach_to_the_survival'>[45]</a>, <a href='#detection_of_informative_features_to_evaluate_the_technical_condition_of_the_transmission_sites_of_mobile'>[47]</a>, <a href='#estimation_of_noise_immunity_for_the_diagnostics_method_for_rotor_units_of_machines_depending_on'>[48]</a>, <a href='#evaluation_of_applicability_of_vibro-sensors_embedded_in_the_modern_mobile_platform_for_tasks_related_to'>[49]</a>, <a href='#vibration_diagnostics_systems_using_mobile_platforms_a_v_tsurko_i_g_davydov_modern_means_of'>[52]</a>, <a href='#comparative_study_of_adaptive_algorithms_for_vibration_diagnostics_of_rolling_bearing_defects_s_yu_borisenko'>[53]</a>, <a href='#wavelet_model_decomposition_of_signal_envelopeа_i_g_davydov_i_m_seglyuk_a_v_tsurko'>[54]</a>, <a href='#models_of_vibration_signals_of_rolling_bearings_in_the_presence_of_surface_defects_i_g'>[56]</a>, <a href='#evaluation_of_sampling_frequencies_of_pulse_processes_using_modifications_of_wavelet_transform_i_g_davydov'>[58]</a>, <a href='#computing_means_of_radio_systems_laboratory_practicum_study_guide_s_yu_vasyukevich_i_g_davydov'>[61]</a>, <a href='#diagnostics_of_industrial_equipment_by_vibration_signals_based_on_periodic_wavelet_transform_research_report_conclusion'>[62]</a>, <a href='#methods_and_means_of_digital_signal_processing_laboratory_workshop_in_2_p_p_1_methods'>[63]</a>, <a href='#device_for_preliminary_phase_processing_of_vibroacoustic_signals_on_pld_i_g_davydov_m_v'>[67]</a>.</p>"
```

Comment:
<p>In VibroBox Alexander develops the expert system for vibration diagnostics. He also works on the equipment which records vibration signals and telemetry data. His current research includes adaptive data analysis (using digital signal processing), data science and artificial intelligence.</p> <p>He has more than five years of experience in many projects, including embedded digital systems design and programming, DSP and telemetry, FPGA, satellite communication technologies, machine learning, cybernetics. He has also led a team of developers and researchers.</p> <p>In 2012 he graduated from Belarusian State University of Informatics and Radioelectronics with a bachelor's degree in Radioinformatics engineering. Later, in 2013, he received a master's degree in Technical Sciences. The title of his thesis is <cite>Adaptive algorithms for early detection of rolling bearing defects by vibration signals</cite>.</p> <p>Now he is a postgraduate student and prepares to defend his Ph.D. thesis in System Analysis.</p> <p>Alexander is a research scientist and assistant of Information Radiotechnologies Department at BSUIR. His interests include the future of humanity and sustainable development of culture and nature. The most favorite free time activities are walking, backpacking, cycling, sailing.</p>